data/help.txt

```diff
@@ -103,18 +103,18 @@ help "navigation"
 	`When you reach a new system, you can press <Land on planet / station> to land on any inhabited planets that are there.`
 	`Also, don't worry about crashing into asteroids or other ships; your ship will fly safely below or above them.`
 
-help "outfitter"
+help "outfitter basics"
```

Comment:
Won't renaming those retrigger the tutorial for everybody?

Reply:
Lots of people seem to have missed it, so...

Reply:
we could keep this one named outfitter but many ppl def missed the other 2

Reply:
is it ok if the 2 advanced tutorials trigger again? I think it is honestly